### PR TITLE
Fix broken links in 2nd kata tutorial Downsampling

### PR DESCRIPTION
**Pull request by AdamDomagalsky**
Signed-off-by: Adam Domagalski <adadom@st.amu.edu.pl>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

2 links fixes
## Verification

Run the tuto 🤡

### CODE REVIEW

File: tutorials/katacoda/thanos/2-lts/step1.md [hunk 1]
```diff
@@ -144,6 +144,6 @@ Similar to previous course let's check if the Querier works as intended. Let's l
 
 This should list the sidecar, including the external labels.
 
-On graph you should also see our 5 series for 1y time, thanks to Prometheus and sidecar StoreAPI: [Graph](https://[[HOST_SUBDOMAIN]]-9091-[[KATACODA_HOST]].environments.katacoda.com/https://2886795307-9091-ollie02.environments.katacoda.com/graph?g0.range_input=1y&g0.max_source_resolution=0s&g0.expr=continuous_app_metric0&g0.tab=0).
+On graph you should also see our 5 series for 1y time, thanks to Prometheus and sidecar StorAPI: [Graph](https://[[HOST_SUBDOMAIN]]-9091-[[KATACODA_HOST]].environments.katacoda.com/graph?g0.range_input=1y&g0.max_source_resolution=0s&g0.expr=continuous_app_metric0&g0.tab=0).
```
Comment (prmsrswt) [inline tutorials/katacoda/thanos/2-lts/step1.md:147]: `StoreAPI` -> `StorAPI` seems unintentional.

**Review by prmsrswt — COMMENTED** [review]
Just a small change, otherwise LGTM. Thanks!

**Comment by prmsrswt** [comment]
Looks like there is a conflict, can you rebase this on `master`? :smile:

**Comment by AdamDomagalsky** [comment]
#3702 already fixed - although I was faster 🕺

**Pull request closed without merging**